### ISSUE TITLE
Remove pre-made ammo when starting with ruined firearms

### ISSUE THREAD

**Issue by ReislerArt** [DigitalzombieTLD/CanneryManufacturingDLC#2]
When having firearms start in a ruined state it would also make sense to remove any pre-made ammo and only leaving ammo crafting items (like shells, gunpowder, etc.).

**Comment by onebunchmugen** [comment]
i agree with this idea. İf we are talking about interloper there shouldnt be ammos lying around everywhere. This is too OP for interloper
